libs/sdk-ui-kit/styles/scss/button.scss

```diff
@@ -595,14 +600,14 @@
 
     > .gd-button:first-child,
     > *:not(.gd-button):first-child .gd-button {
-        @include border-radius(3px 0 0 3px);
+        border-radius: $button-group-childs-border-radius;
```

Comment:
this should not be single variable. It should be here like :
 $button-border-radius 0 0 $button-border-radius